src/content/docs/browser/new-relic-browser/troubleshooting/mootools-related-errors.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: MooTools related errors encountered
+type: troubleshooting
+tags:
+  - Browser
+  - Browser monitoring
+  - Troubleshooting
+metaDescription: 'If you use the MooTools library, you may need to update to the nocompat build of MooTools version 1.6.0 to address conflicts.'
+---
+
+## Problem
+
+You're using the MooTools library and notice errors like the ones below showing up in the browser console or on the [**JavaScript errors** page](/docs/browser/new-relic-browser/browser-pro-features/js-errors-dashboard-examining-errors-over-time).
+
+- `Uncaught TypeError: t is not a function`
+
+- `Uncaught (in promise) TypeError: Failed to execute 'removeEventListener' on 'EventTarget': 2 arguments required, but only 0 present.`
+
+## Cause
+
+The MooTools library (especially its compatibility layer) mutates a number of native JavaScript objects and methods, which can cause conflicts with newer libraries, including the New Relic browser agent.
+
+## Solutions
+
+1. Because the last version of MooTools was released in 2016, migrating from MooTools to other actively maintained libraries may be the best strategic choice when possible.
```

Comment:
```suggestion
1. Because the last version of MooTools was released in 2016, we recommend migrating from MooTools to other actively maintained libraries if possible.
```